packages/indexer-common/src/indexer-management/monitor.ts

```diff
@@ -610,15 +610,43 @@ export class NetworkMonitor {
         )
       }
 
-      // Check for validity with epoch manager currentEpoch, and fetch block hash
-      const validBlock =
-        result.data.network.latestValidBlockNumber.epochNumber ==
-        (await this.currentEpochNumber())
-          ? result.data.network.latestValidBlockNumber
-          : result.data.network.latestValidBlockNumber.previousBlockNumber
-
-      // Resolve block hash for the given block number.
-      // Calls the configured provider for blocks from protocol chain, or Graph Node otherwise.
+      // Check for validity with Epoch Manager's currentEpoch method, and fetch block hash after that.
+      const epochManagerCurrentEpoch = await this.currentEpochNumber()
+      const epochSubgraphCurrentEpoch =
+        result.data.network.latestValidBlockNumber.epochNumber
+
+      // The Epoch Manager will always lead new epochs, so it should be minuend.
+      const epochDifference = epochManagerCurrentEpoch - epochSubgraphCurrentEpoch
+
+      if (epochDifference < 0) {
+        // Somehow, the Epoch Manager is behind the Epoch Subgraph. This is a critical failure.
+        const criticalErrorMessage = `The Epoch Manager is behind the Epoch Subgraph`
+        this.logger.critical(criticalErrorMessage, {
+          epochManagerCurrentEpoch,
+          epochSubgraphCurrentEpoch,
+        })
+        throw new Error(criticalErrorMessage)
+      }
+
+      if (epochDifference > 1) {
+        // It is not acceptable for the Epoch Subgraph to be more than one epoch away from the
+        // Epoch Manager.
```

Comment:
In this context what's important is meeting the arbitration charter, so to make that clear I'd update the comment slightly: 
"It is not acceptable to submit a POI for an epoch more than one epoch away from the current epoch

Reply:
Fixed in 65380140